android/app/src/main/java/app/priceguard/ui/home/mypage/MyPageSettingAdapter.kt

```diff
@@ -32,6 +34,12 @@ class MyPageSettingAdapter(
             with(binding) {
                 settingItemInfo = item
                 listener = clickListener
+
+                if (item.id == Setting.DELETE_ACCOUNT) {
+                    val typedValue = TypedValue()
+                    binding.root.context.theme.resolveAttribute(R.attr.colorError, typedValue, true)
```

Comment:
binding scope 내부에 있는 코드라 binding을 다시 사용할 필요는 없어 보입니다.

```suggestion
                    this.root.context.theme.resolveAttribute(R.attr.colorError, typedValue, true)
```

Reply:
그러네요.. 제거했습니다!